torch/utils/data/dataloader.py

```diff
@@ -366,7 +380,7 @@ class DataLoader(object):
 
     def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None, batch_sampler=None,
                  num_workers=0, collate_fn=default_collate, pin_memory=False, drop_last=False,
-                 timeout=0, worker_init_fn=None):
+                 timeout=0, worker_init_fn=None, ind_worker_queue=False):
```

Comment:
Could you add a check that raise ValueError if this flag is set and `num_workers<=0`?

Reply:
Done.